package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "blockchain-core",
+  "version": "1.0.0",
+  "license": "GPL-3.0",
+  "description": "Implementation of core blockchain with Javascript",
+  "main": "index.js",
+  "repository": "https://github.com/Zigurat-blockchain-masters/blockchain-core.git",
+  "contributors": [
+    {
+      "name": "Roman Kokhatskyi",
+      "github": "rkokhatskyi"
+    },
+    {
+      "name": "Jean-Christophe Rigo",
+      "github": "jcrigo86"
+    },
+    {
+      "name": "Walter von Gabain",
+      "github": "waltervongabain"
+    },
+    {
+      "name": "Miguel Santamaria",
+      "github": "rkokhatskyi"
+    },
+    {
+      "name": "Roman Kokhatskyi",
+      "github": "rkokhatskyi"
+    }
+  ],
```

Comment:
Not that cool leaving me out!

"name" : "Angelos Nistikoulis",
"github": "Agn21"